pkg/controller/appdefinition/deploy.go

```diff
@@ -867,3 +868,10 @@ func ToDeployments(req router.Request, appInstance *v1.AppInstance, tag name.Ref
 
 	return result, nil
 }
+
+func stringOrNilPtr(str string) *string {
+	if str != "" {
+		return z.Pointer[string](str)
```

Comment:
nit: necessary?
```suggestion
		return z.Pointer(str)
```

Reply:
If I understand the signature: https://github.com/acorn-io/z/blob/main/z.go#L21 and the usage on line 782:                                         
```
...
TerminationGracePeriodSeconds: z.Pointer[int64](10),
...
```

I believe it is necessary.